circuit/src/utils.rs

```diff
@@ -291,3 +292,46 @@ where
 
 	Ok(res)
 }
+
+/// The u64 integer represented by an L-bit little-endian bitstring.
+///
+/// # Panics
+///
+/// Panics if the bitstring is longer than 64 bits.
+pub fn lebs2ip<const L: usize>(bits: &[bool; L]) -> u64 {
```

Comment:
Can we call this `bits_to_u64` to stay consistent

Reply:
I add slight modification. 
I renamed `lebs2ip` to `le_bits_to_u64`.
I think `le` should be added for clearer name.
https://github.com/eigen-trust/protocol/pull/169/commits/ab695818a4f077629984c7e35e847639235712ca